Source/MoreDotNet.Test/Extensions/Common/RandomExtensions/NextCharTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿namespace MoreDotNet.Tests.Extensions.Common.RandomExtensions
+{
+    using System;
+
+    using Moq;
+    using MoreDotNet.Extensions.Common;
+
+    using Xunit;
+
+    public class NextCharTests
+    {
+        [Fact]
+        public void NextChar_ShouldReturnChar()
+        {
+            var random = new Random();
+            var result = random.NextChar();
+
+            Assert.IsType<char>(result);
```

Comment:
This check is a but pointless? Can you think of a scenario where the NextChar() method wont return a Char? The return type is guaranteed by the method signature.